packages/odyssey-react-mui/src/labs/DatePickers/DateTimePicker.tsx

```diff
@@ -39,6 +39,10 @@ import {
 } from "../../OdysseyDesignTokensContext.js";
 import { OdysseyThemeProvider } from "../../OdysseyThemeProvider.js";
 
+export const DateTimePickerSize = {
+  maxWidth: "34.6974rem", // ~485.75px
+};
```

Comment:
Since this isn't coming from design tokens, I wrote a `pxToRem` function that can take your `px` value and convert to `rem` for you:
![image](https://github.com/user-attachments/assets/d0a9256f-be41-4fc2-9f40-5eacc7851866)

`maxWidth: pxToRem(486)rem`